build.gradle.kts

```diff
@@ -1,6 +1,15 @@
 plugins {
     kotlin("jvm") version "2.0.10"
     kotlin("plugin.serialization") version "1.9.21"
```

Comment:
Different versions for the Kotlin jvm plugin (2.0.10) and the Kotlin serialization plugin (1.9.21). These versions should match to avoid potential incompatibility issues.